### ISSUE TITLE
Unable to resolve suitable regular or attached property Mode on type Avalonia.Themes.Fluent.

### ISSUE THREAD

**Issue by cyraid** [AvaloniaUI/Avalonia#10414]
**Describe the bug**
It does not build. I installed nuget "Avalonia.Themes.Fluent"
`<FluentTheme Mode="Light"/>`

**Expected behavior**
Very simple app, should be running. If I remove the xml line from App.axaml, the app will run having no background window or anything.

**Desktop (please complete the following information):**
- OS: Windows 11

**Comment by maxkatz6** [comment]
https://github.com/AvaloniaUI/Avalonia/wiki/Breaking-Changes#avaloniathemesfluent-and-simple

**Comment by timunie** [comment]
In 11.0 preview5 please remove Mode from FluentTheme. 

<FluentTheme /> still works. 

If you want to switch Light or Dark theme, use App.RequestedThemeMode instead.

**Comment by cyraid** [comment]
Removing it doesn't draw anything.
![image](https://user-images.githubusercontent.com/7562753/220202017-70be5b6f-7ec4-42f5-92b1-48475159709a.png)

**Comment by maxkatz6** [comment]
FluentTheme shouldn't be removed.

**Comment by cyraid** [comment]
> FluentTheme shouldn't be removed. Only a Mode property.

Correct. The XML tag is the only thing I removed. Not the FluentTheme package.

**Comment by cyraid** [comment]
![image](https://user-images.githubusercontent.com/7562753/220202830-c22a03f7-cdd6-4ca1-bf85-0920202f5721.png)

**Comment by jp2masa** [comment]
You should still have `<FluentTheme />` in your `App.xaml`, i.e. only remove `Mode="..."` part.

**Comment by cyraid** [comment]
> You should still have `<FluentTheme />` in your `App.xaml`, i.e. only remove `Mode="..."` part.

Thank you @jp2masa ! That works. :D

**Comment by kazeens** [comment]
Update docs
https://docs.avaloniaui.net/docs/next/tutorials/music-store-app/creating-a-modern-looking-window
Cause in docs it doesn't work

**Comment by timunie** [comment]
@kazeens thx for pointing that out. A hotfix PR is out. However, if you see such things in the docs (in the future), a PR to fix them is very welcome 😊.

**Comment by thscott** [comment]
@timunie I can't seem to see a PR anywhere fixing that issue (and the document still currently uses `Mode="Dark"` [here](https://github.com/AvaloniaUI/avalonia-docs/blob/main/docs/tutorials/music-store-app/creating-a-modern-looking-window.md)).

**Comment by timunie** [comment]
@kazeens okay, the reason is it was changed on another page but not on the tutorial. I wasn't aware of this. Thx for pointing this out.
